docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -89,6 +97,7 @@ export computes=(
   # ...
 )
 ----
++
 ** Replace `["standalone.localdomain"]="192.168.122.100"` with the name of the Compute node and its IP address.
```

Comment:
I do the callouts like <1> https://docs.asciidoctor.org/asciidoc/latest/verbatim/callouts/

Maybe: "Replace `["standalone.localdomain"]="192.168.122.100"` with the name and IP address of the Compute node."